internal/benchmark/bench.cu

```diff
@@ -5,6 +5,8 @@
 #include <thrust/reduce.h>
 #include <thrust/scan.h>
 
```

Comment:
Let's not have a newline between `<thrust/detail/config.h>` and the other Thrust headers.